code/datums/status_effects/debuffs.dm

```diff
@@ -968,3 +968,17 @@
 /datum/status_effect/cloudstruck/Destroy()
 	. = ..()
 	QDEL_NULL(mob_overlay)
+
+
+/datum/status_effect/ghoul
+	id = "ghoul"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1
+	examine_text = "<span class='warning'>SUBJECTPRONOUN has a blank, lifeless stare.</span>"
```

Comment:
That'll make trying to be stealthy pointless. Right now you can just wear mask with a helmet but this will make it impossible to hide that you're a ghoul. I suggest removing it

Reply:
You're right. It's gone.